msd700_odom:
  wheel_radius    : 2.7   # cm
  wheel_distance  : 23    # cm
  encoder_ppr     : 2400   # Pulse per Rotation
  compute_period  : 10    # ms
  odom_frame      : odom
  robot_frame     : base_link
  debug           : false

